QVRWeekView/Classes/Views/WeekView.swift

```diff
@@ -44,6 +44,21 @@ open class WeekView: UIView {
         let firstActiveDay = self.dayScrollView.activeDay
         return firstActiveDay...(firstActiveDay + Int(LayoutVariables.visibleDays - 1))
     }
+    
+    //A date range containing the current days visible on screen
+    public var visibleDateRange: ClosedRange<Date> {
+        let firstActiveDay = self.dayScrollView.activeDay
+        let lastActiveDay = (firstActiveDay + Int(LayoutVariables.visibleDays - 1))
+        return firstActiveDay.getDateWithTime(hours: 0, minutes: 0, seconds: 0)...lastActiveDay.getDateWithTime(hours: 23, minutes: 59, seconds: 59)
```

Comment:
Very minor, but I think I have already made Date extensions for this type of thing. Could you try out:
```
firstActiveDay.dateObj.getStartOfDay()...lastActiveDay.dateObj.getEndOfDay()
```
and see if that works as well?

Reply:
I don't have those extensions; perhaps I need a branch other than master for those?

Reply:
It is done!